plugin/evm/atomic/vm/tx_gossip_test.go

```diff
@@ -311,18 +311,61 @@ func TestAtomicTxPushGossipInbound(t *testing.T) {
 	require.NoError(err)
 	require.NoError(vm.AtomicMempool.AddLocalTx(tx))
 
-	marshaller := atomic.TxMarshaller{}
-	gossipBytes, err := marshaller.MarshalGossip(tx)
-	require.NoError(err)
+	inboundGossipMsg := buildAtomicPushGossip(t, tx)
+	require.NoError(vm.AppGossip(ctx, ids.EmptyNodeID, inboundGossipMsg))
+	require.True(vm.AtomicMempool.Has(tx.ID()))
+}
+
+// Tests that conflicting txs are handled, based on whether the original tx is
+// valid or not.
+func TestAtomicTxPushGossipInboundConflicting(t *testing.T) {
+	require := require.New(t)
```

Comment:
You don't need this. The only thing it achieves is implicitly passing `testing.T` to the concrete `require` assertion call, but it adds unnecessary noise.